.jsonlintschema

```diff
@@ -12,7 +12,7 @@
             "title": {
               "description": "The icons name",
               "type": "string",
-              "required": true
+              "required": false
```

Comment:
With the update to the `primaryBuild` scripts setting defaults for title, style and desc, having these values as required is no longer true.

Reply:
I disagree. The point of this check is to make sure that the built resources are done correctly. The process was always automated to some extent, your update added an additional feature to that support. 

Please change them back.

Reply:
I can't change it back. When I change it back the linter screams because I have entries in the icons.json that don't have those fields, and depend on the build process to generate them automatically. 

With the linter screaming I'm prevented from being able to make the commit.

Reply:
Ugh ... I get it now. Because you reduced the spec on the data needed to construct the SVG, that is what the impact is here. 

![alt text](https://media.giphy.com/media/fQoxwZBVWq5jhLXRty/giphy.gif)